KsApi/models/Comment.swift

```diff
@@ -66,6 +66,30 @@ extension Comment: Decodable {
   }
 }
 
+extension Comment {
+  public static func optimisticComment(
```

Comment:
Should probably rename this to be a more direct representation of what the method is doing

Reply:
Maybe something like `failableComment`